localnet/manage.sh

```diff
@@ -49,6 +54,8 @@ main() {
     if [ -z $CI ]; then
       export START_TRACKER="y"
       get-workflow
```

Comment:
Can someone double check? I think the `export START_TRACKER="y"` line is useless here, because we reset the variable in `get-workflow` anyways. Right?